packages/react/README.md

```diff
@@ -6,7 +6,7 @@ A set of components for integrating [Knock](https://knock.app) into a React appl
 
 [See a live demo](https://knock-in-app-notifications-react.vercel.app/)
 
-![In-app feed component example](NotificationFeed.png)
+![In-app feed component example](https://github.com/knocklabs/javascript/raw/main/packages/react/NotificationFeed.png)
```

Comment:
The image doesn't load on the NPM package page so I think linking to the actual image instead of relative path should fix it.